### PR TITLE
drop import sort

### PR DESCRIPTION
**Pull request by itajaja**
this should not be merged until a new release of eslint-plugin-import incorporates https://github.com/benmosher/eslint-plugin-import/commit/614e55f8306cddd55066babcae653af9fb9ead92

### CODE REVIEW

**Comment by itajaja** [comment]
the travis error is specifically because of that unsolved issue

File: packages/babel-preset/index.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 /* eslint-disable no-console */
 
-const { loadConfig } = require('browserslist');
-const pick = require('lodash/pick');
 const path = require('path');
 const envPreset = require('@babel/preset-env');
 const reactPreset = require('@babel/preset-react');
-
+const { loadConfig } = require('browserslist');
```
Comment (taion) [inline packages/babel-preset/index.js]: @itajaja what's happening here?

Comment (itajaja) [reply packages/babel-preset/index.js]: system imports (`path` here) are grouped first.

File: packages/eslint-config-4catalyzer/rules.js [hunk 2]
```diff
@@ -78,4 +84,13 @@ module.exports = {
     },
   ],
   'import/prefer-default-export': 'off',
+  'import/order': [
+    'error',
+    {
+      'newlines-between': 'never',
```
Comment (itajaja) [inline packages/eslint-config-4catalyzer/rules.js]: @taion and @jquense let's forget for a moment this is broken due to issue above... do you think should be `always` or `always-inside-groups` ([doc](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#newlines-between-ignorealwaysalways-and-inside-groupsnever))

Comment (taion) [reply packages/eslint-config-4catalyzer/rules.js]: i think we want newlines between groups, but not within groups?

Comment (itajaja) [reply packages/eslint-config-4catalyzer/rules.js]: so `always`? sure. yeah. if there is a reason to override, is ok to `eslint-ignore`

**Comment by taion** [comment]
is this gtg?

File: packages/eslint-config-4catalyzer-react/test/config.test.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 const pick = require('lodash/pick');
+
 const configTester = require('../../../tools/config-tester');
+
```
Comment (jquense) [inline packages/eslint-config-4catalyzer-react/test/config.test.js]: what is the grouping here that this happens?

File: packages/eslint-config-4catalyzer/rules.js [hunk 4]
```diff
@@ -80,4 +86,13 @@ module.exports = {
     },
   ],
   'import/prefer-default-export': 'off',
+  'import/order': [
```
Comment (taion) [inline packages/eslint-config-4catalyzer/rules.js]: can we set `groups` to just `[['builtin', 'external']]`? or maybe `['builtin', 'external']` if you prefer?

**Comment by itajaja** [comment]
> is this gtg?

on my end!

File: packages/eslint-config-4catalyzer/rules.js (hunk 4 above)
Comment (taion) [reply packages/eslint-config-4catalyzer/rules.js]: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array

Comment (taion) [inline packages/eslint-config-4catalyzer/rules.js]: do we want case-insensitivity in the ordering? https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#alphabetize-order-ascdescignore-caseinsensitive-truefalse

File: packages/eslint-config-4catalyzer-react/test/config.test.js (hunk 3 above)
Comment (itajaja) [reply packages/eslint-config-4catalyzer-react/test/config.test.js]: I think parent imports are treated differently from sibling imports. not sure if it's a feature or a bug

**Comment by taion** [comment]
and that _is_ a question. i'd be fine w/breaking things out into more groups. not sure about having newly-sep between parents, siblings, &c. though

File: packages/eslint-config-4catalyzer-react/test/config.test.js (hunk 3 above)
Comment (taion) [reply packages/eslint-config-4catalyzer-react/test/config.test.js]: see the groups option

**Comment by itajaja** [comment]
ok, I think the `..` is probably a bug. with these groups: `["builtin", "external", ["parent", "sibling", "index"]]`

I get this:

```
const a = require('.');
const b = require('../..');
const g = require('../../..');
const c = require('../../../tools/config-tester');
const f = require('../a');
const d = require('./a');

const e = require('..');
```

I'd be ok with this for now, considering it's such a small corner case

**Comment by itajaja** [comment]
g2g on my end. [this](https://github.com/4Catalyzer/javascript/pull/539/files#diff-c7f332aacac6647d6f8e06c67ae4713cR1) is clearly a bug but I think it's so small that it's ok to merge this

**Comment by jquense** [comment]
Let's try it out. We can always switch it up

**Comment by taion** [comment]
this probably should have been marked "breaking"

**Pull request merged**